pubspec.yaml

```
@@ -17,26 +17,26 @@ environment:
   # https://github.com/dart-lang/stagehand/issues/617
   #
   # Also make sure this minimum sdk version is reflected in `.travis.yml`.
-  sdk: '>=2.10.0 <3.0.0'
+  sdk: ">=2.10.0 <3.0.0"
 
 # Add the bin/stagehand.dart script to the scripts pub installs.
 executables:
   stagehand:
 
 dependencies:
-  args: ^1.5.0
-  http: ^0.12.0
+  args: ^2.0.0
+  http: ^0.13.0
   path: ^1.7.0
```

```suggestion
  path: ^1.8.0
```

If we want these deps to be null safe we need to bump the minimum bound for the ones without major version changes as well.

---

Stagehand itself doesn't need to be migrated to use null safety (we're soon moving the templates to the `dart` tool and discontinuing stagehand). It just needs to create output that supports null safety.

---

Done